setup.py

```diff
@@ -48,6 +48,9 @@ def get_version():
     author='Ian Cordasco, Cory Benfield',
     author_email="graffatcolmingov@gmail.com",
     url="https://toolbelt.readthedocs.io/",
+    project_urls={
+        "Source": "https://github.com/requests/toolbelt",
```

Comment:
Would be nice to have changelog as well:
```suggestion
        "Changelog": "https://github.com/requests/toolbelt/blob/master/HISTORY.rst",
        "Source": "https://github.com/requests/toolbelt",
```